setup.py

```diff
@@ -6,15 +6,15 @@
 setuptools.setup(
     name="gradient-accumulator",
     version="0.5.2",
-    author="André Pedersen and David Bouget and Javier Pérez de Frutos and Tor-Arne Schmidt Nordmo",
+    author="André Pedersen and Derek Alexander and David Bouget and Javier Pérez de Frutos and Tor-Arne Schmidt Nordmo",
     author_email="andrped94@gmail.com",
     description="Package for gradient accumulation in TensorFlow",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/andreped/GradientAccumulator",
     packages=setuptools.find_packages(exclude=('tests', 'notebooks', 'assets', 'docs', 'shell')),
     install_requires=[
-        "tensorflow",
+        "tensorflow<=2.10.0",
```

Comment:
If you want to put something here, it should be `tensorflow>=2.2`. Right now tests for Python 3.11 fail as there is no precompiled TF binary for `<=2.10`.

Reply:
I've reverted this change for now, to be addressed in a subsequent PR. I suggest some level of version control, otherwise we might get stuck needed to wire up the CI to test for all TF versions where the optimizer API changed...